esp32_s3_lcd_ev_board/esp32_s3_lcd_ev_board.c

```diff
@@ -20,6 +20,7 @@
 #include "lvgl.h"
 
 #include "bsp/esp32_s3_lcd_ev_board.h"
+#include "bsp/display.h"
```

Comment:
Here is missing implementation of the `bsp_display_new`

Reply:
I'll leave the LCD_EV board for follow-up MR. I don't understand the RGB driver enough, yet.